util/src/main/java/com/graphicsfuzz/util/Constants.java

```diff
@@ -23,6 +23,7 @@ private Constants() {
   }
 
   public static final String GLF_COLOR = "_GLF_color";
+  public static final String GLF_FRAGCOORD = "_GLF_FragCoord";
```

Comment:
It would be good to add a comment before this to mention that it is specifically used to model gl_FragCoord in a tool that converts a fragment shader into a vertex shader.

Reply:
Added